deployment/bin/check_abandoned_containers.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
I didn't look carefully at this script but since it's not used by the workers it should be fine.

Reply:
@kkellerlbl It's being used in start-condor.sh